dkist/net/helpers.py

```diff
@@ -0,0 +1,95 @@
+"""
```

Comment:
Not to be That Guy, but "helpers.py" is a guaranteed terrible name for a module.